### PR TITLE
Added some tests for print_board

### PR DESCRIPTION
**Pull request by matthew-wood-bjss**
Other: added some tests for print_board
 -> also added a check to not run the game if it is imported as a module

Does not impact the rest of the project; check statement will ensure tictactoe game starts only when file is run.

Working tests for print_board, which prints the current board state, given the board.

88% test coverage, and passes all unit tests

### CODE REVIEW

File: tictactoetest/tictactoetest/tictactoe.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ def is_winner(board,isplayer1):
         return False
             
 def print_board(board):
```
Comment (JamesExeter) [inline tictactoetest/tictactoetest/tictactoe.py]: why haven't you type hinted this?

**Review by JamesExeter — CHANGES_REQUESTED** [review]
please add type hints

File: tictactoetest/tictactoetest/tictactoe.py (hunk 1 above)
Comment (matthew-wood-bjss) [reply tictactoetest/tictactoetest/tictactoe.py]: OMG! I will do this :)

Comment (matthew-wood-bjss) [reply tictactoetest/tictactoetest/tictactoe.py]: All done :)

**Pull request merged**